contributors/guide/first-contribution.md

```diff
@@ -19,57 +19,96 @@ description: |
-Those interested in contributing without writing code may also find ideas in the [Non-Code Contributions Guide](non-code-contributions.md).
+The first step to getting starting contributing to Kubernetes is to find something
+to work on. Help is always welcome, and no contribution is too small! 
+
+Here are some things you can do today to get started contributing:
+
+* Help improve the Kubernetes documentation
+* Clarify code, variables, or functions that can be renamed or commented on
+* Write test coverage
+* Help triage issues
+
+If the above suggestions don't appeal to you, you can browse the 
+[Contributor Role Board] to see who is looking for help.
+
+Those interested in contributing without writing code can also find ideas in the 
+[Non-Code Contributions Guide].
 
 ### Find a good first topic
 
-There are [multiple repositories](https://github.com/kubernetes/) within the Kubernetes organization.
-Each repository has beginner-friendly issues that provide a good first issue.
-For example, [kubernetes/kubernetes](https://git.k8s.io/kubernetes) has [help wanted](https://go.k8s.io/help-wanted) and [good first issue](https://go.k8s.io/good-first-issue) labels for issues that should not need deep knowledge of the system.
-The `good first issue` label indicates that members have committed to providing [extra assistance](/contributors/guide/help-wanted.md) for new contributors.
+There are [multiple repositories] within the Kubernetes organization.
+Each repository has beginner-friendly issues that are a great place to 
+get started on your contributor journey.
+
+For example, [kubernetes/kubernetes] has [help wanted] and [good first issue] 
+labels for issues that don't need high-level Kubernetes knowledge to contribute to.
+
+The `good first issue` label also indicates that Kubernetes Members have 
+committed to providing [extra assistance] for new contributors.
 
-Another good strategy is to find a documentation improvement, such as a missing/broken link, which will give you exposure to the code submission/review process without the added complication of technical depth. Please see [Contributing](#contributing) below for the workflow.
+Another way to get started is to find a documentation improvement, such as a 
+missing/broken link, which will give you exposure to the code submission/review 
+process without the added complication of technical depth. 
 
-#### Issue Assignment in Github
+### Issue Assignment in Github
 
-When you are willing to take on an issue, you can assign it to yourself. Just reply with `/assign` or `/assign @yourself` on an issue, 
-then the robot will assign the issue to you and your name will present at `Assignees` list.
+When you've found an issue to work on, you can assign it to yourself.
+
+* Reply with `/assign` or `/assign @yourself` on the issue you'd like to work on 
+* The [K8s-ci-robot] will automatically assign the issue to you. 
+* Your your name will then be listed under, `Assignees`.
 
 ### Learn about SIGs
```

Comment:
```suggestion
## Learn about SIGs
```
Hmmm....Thinking about it, this might be worth bumping back out to an H2 and adjusting the table of contents. That or potentially removing the first heading because everything in the doc is sort of a child item under it.